frontend-react/src/content/support/index.ts

```diff
@@ -8,15 +8,17 @@ import {
     ContentDirectory,
     ElementDirectory,
 } from "../../components/Content/MarkdownDirectory";
-import { Contact, Faq } from "../../pages/support/index-legacy";
+import { Contact, ServiceRequest, Faq } from "../../pages/support/index-legacy";
 
 enum SupportTitles {
     CONTACT = "Contact",
+    SERVICEREQUEST = "Service request form",
```

Comment:
(Non-blocking) For enum names with more than one word, we separate them with an underscore

Reply:
Good catch! Updated in my last commit!